### PR TITLE
Fixes #1435 - MNI (Multi-network-interior)

### LINKED ISSUE #1435: Multi-Network Interior feature
This feature allows an interior network of routers to provide multi-tenant access for multiple VANs comprised of attached edge routers.

Each edge router is configured with a van-id.  The edges that share the same van-id are part of a Skupper VAN and are isolated from other edge routers that use different van-ids.

### PR DESCRIPTION
**Pull request by ted-ross**
Added optional van_id to router configuration and management. Add van_id to temporary addresses when present.
Annotate adaptor addresses with the van-id.

### CODE REVIEW

**Comment by ted-ross** [comment]
Before merging, a system test for this feature should be added.

**Review by ganeshmurthy — APPROVED** [review]

**Review by kgiusti — APPROVED** [review]
LGTM, modulo G-man's comments

**Pull request merged**
